app/index.js

```diff
@@ -152,31 +154,27 @@ module.exports = yeoman.generators.Base.extend({
 			{
 				buildFormat: this.buildFormat,
 				componentName: this.componentName,
-				templateLanguage: this.templateLanguage
+				templateLanguage: this.templateLanguage,
+				kebabCaseName: this.kebabCaseName
 			}
 		);
 		this.fs.copyTpl(
-			this.templatePath('test/_Boilerplate.js'), this.destinationPath('test/' + this.componentName + '.js'),
+			this.templatePath('__tests__/_Boilerplate.js'), this.destinationPath('__tests__/' + this.componentName + '.js'),
```

Comment:
mmm... it makes sense for us as maintainers, but I doubt that if you pick Karma as a test environment you expect tests to be there...